src/network/connection.h

```diff
@@ -107,28 +107,25 @@ class Connection {
 	);
 #endif
 
-	const ServerFeatures& getServerFeatureFlags() {
-		return m_ServerFeatures;
-	}
+	const ServerFeatures& getServerFeatureFlags() { return m_ServerFeatures; }
 
 	bool beginBundle();
 	bool endBundle();
 
 private:
-	void updateSensorState(std::vector<Sensor *> & sensors);
+	void updateSensorState(std::vector<Sensor*>& sensors);
 	void maybeRequestFeatureFlags();
 
 	bool beginPacket();
 	bool endPacket();
 
-	size_t write(const uint8_t *buffer, size_t size);
-	size_t write(uint8_t byte);
+	size_t write(const uint8_t* buffer, size_t size);
 
 	bool sendPacketType(uint8_t type);
 	bool sendPacketNumber();
 	bool sendFloat(float f);
 	bool sendByte(uint8_t c);
-	bool sendShort(int16_t i);
+	bool sendU16(uint16_t c);
 	bool sendInt(int32_t i);
 	bool sendLong(int64_t l);
```

Comment:
i think this should be uint64_t as the m_PacketNumber is the only thing that is send with that. see above

Reply:
I think you're looking at the wrong thing? I only changed `sendShort()`, not `sendLong()`.